events/religion_events/religious_decision_events.txt

```diff
@@ -1525,16 +1694,42 @@ religious_decision.0302 = {
 	desc = {
 		triggered_desc = {
 			trigger = {
+				NOR = {
+					like_zoroastrianism_religion_trigger = yes
+					like_zunism_religion_trigger = yes
+				}
 			}
 			desc = religious_decision.0302.desc.wilderness
 		}
+		triggered_desc = {
+			trigger = {
+				OR = {
+					like_zoroastrianism_religion_trigger = yes
+					like_zunism_religion_trigger = yes
+				}
+			}
+			desc = religious_decision.0302.desc.zoroastrian
```

Comment:
`religious_decision.0302.desc.zoroastrian:0 "In the interest of avoiding contamination by the corpse demon Nasu, I am not permitted in the Dakhma Tower where`

Reply:
Commented in d2a3fad024b321a0d07baca25219005366da017e.